ai_diffusion/settings.py

```diff
@@ -217,9 +223,40 @@ class Settings(QObject):
 
     save_image_metadata: bool
     _save_image_metadata = Setting(
-        _("Save Image Metadata"),
+        _("Save Image Metadata (PNG)"),
```

Comment:
Maybe in the settings UI, disable this setting when something other than PNG is selected.

I'd keep the name and description unchanged in that case, makes it less likely to require retranslation.